frontend/public/components/start-guide.tsx

```diff
@@ -1,130 +1,131 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
-import { connect } from 'react-redux';
-import { Link } from 'react-router-dom-v5-compat';
-import { Button, Hint, HintTitle, HintBody } from '@patternfly/react-core';
-import { useTranslation, Trans } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { Button, ButtonVariant, Divider, EmptyStateVariant } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
 
 import { FLAGS } from '@console/shared/src/constants';
 import { useActiveNamespace } from '@console/shared/src/hooks/useActiveNamespace';
 import { useActivePerspective } from '@console/dynamic-plugin-sdk';
-import { createProjectMessageStateToProps } from '../reducers/ui';
-import { Disabled, ExternalLink, openshiftHelpBase, LinkifyExternal } from './utils';
-import { connectToFlags } from '../reducers/connectToFlags';
+import { Disabled, openshiftHelpBase, LinkifyExternal, ConsoleEmptyState } from './utils';
 import { ProjectModel } from '../models';
 import { K8sResourceKind } from '../module/k8s/types';
 import { useCreateNamespaceOrProjectModal } from '@console/shared/src/hooks/useCreateNamespaceOrProjectModal';
+import { RootState } from '../redux';
+import { useFlag } from '@console/shared/src';
+import { ClusterIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
 
-export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)(
-  ({ canCreate = true, createProjectMessage }: OpenShiftGettingStartedProps) => {
-    const { t } = useTranslation();
-    const [, setActiveNamespace] = useActiveNamespace();
-    const [perspective] = useActivePerspective();
-    const createNamespaceOrProjectModal = useCreateNamespaceOrProjectModal();
+export const OpenShiftGettingStarted: React.FCC<OpenShiftGettingStartedProps> = () => {
+  const { t } = useTranslation();
+  const [, setActiveNamespace] = useActiveNamespace();
+  const [perspective] = useActivePerspective();
+  const canCreateNamespace = useFlag(FLAGS.CAN_CREATE_NS);
+  const canCreateProject = useFlag(FLAGS.CAN_CREATE_PROJECT);
+  const canCreate = canCreateNamespace || canCreateProject;
+  const createProjectMessage = useSelector(({ UI }: RootState) => UI.get('createProjectMessage'));
+  const createNamespaceOrProjectModal = useCreateNamespaceOrProjectModal();
+  const onClickCreate = () =>
+    createNamespaceOrProjectModal({
+      onSubmit:
+        perspective !== 'admin'
+          ? (project: K8sResourceKind) => {
+              setActiveNamespace(project.metadata?.name);
+            }
+          : undefined,
+    });
+
+  const primaryActions = canCreate
+    ? [
+        <Button key="create-project-action" variant={ButtonVariant.primary} onClick={onClickCreate}>
+          {t('public~Create a new Project')}
```

Comment:
now I wonder if we should reduce the i18n effort a bit by reusing an existing string

```suggestion
          {t('public~Create {{label}}', { label: "Project" })}
```